src/package.json

```diff
@@ -19,11 +19,8 @@
         "plugin.link": "npm link && cd ../demo && npm link nativescript-yourplugin && cd ../src",
         "plugin.tscwatch": "npm run tsc -- -w",
         "demo.ios": "npm i && npm run tsc && cd ../demo && tns run ios --syncAllFiles",
-        "demo.android": "npm i && npm run tsc && cd ../demo && tns run android --syncAllFiles"
-    },
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/YourName/nativescript-yourplugin.git"
+        "demo.android": "npm i && npm run tsc && cd ../demo && tns run android --syncAllFiles",
+        "nuke": "rm -rf node_modules && cd ../demo && rm -rf hooks node_modules platforms"
```

Comment:
Would mind to rename the command from "nuke" to "clean" or "clear"? I think it would be more meaningful for users. What's your opinion?